app/src/components/common/FullPageHeader/index.tsx

```diff
@@ -12,13 +13,14 @@ interface HeaderProps {
 
 export const FullPageHeader: React.FC<HeaderProps> = ({ showUserHeader = false, doRedirectOnLogoClick = false }) => {
   const navigate = useNavigate();
+  const isBrowserStackIntegrationEnabled = useFeatureIsOn("browserstack_integration");
```

Comment:
Can we extract this into `useIsBrowserstackIntegrationOn` custom hook which will be a common point where we need to change this condition. 

In future we need to add `localStorage.get("forceBsIntegration")` flag too using the bookmarklet. So this will make it easy to change the condition

Reply:
Also we should consume this hook everywhere. Auth, pricing ...